### ISSUE TITLE
I cannot add/edit a contact since upgrading to 2.15.0

### ISSUE THREAD

**Issue by dsalvat-cells** [mautic/mautic#7243]
## Bug Description

Since we've upgraded from 2.14.1 to 2.15.0 we've been unable to create or update any contact. Strange enough, we have a subscribe form which works perfectly fine. 

I've also tried to do a fresh install of the 2.15.0, just in case the upgrade did not go as expected and using a copy of the DB. We had the same results.

| Q   | A
| --- | ---
| Mautic version | 2.15.0
| PHP version | 7.0.30
| Browser | Chrome 72 and Firefox 65

### Steps to reproduce
1. Go to the Contacts option.
2. Click on any Contact.
3. Select the "Edit" button.
4. Change the name of the contact and click on "Save & Close" or even on "Apply"
5. The blue line moves a bit and then nothing happens.
6. Closing and opening again the same contact edition page, will show that no changes have been applied
 
### Log errors
This is what we get from the app/logs/prod-2019-XXXXX.php file,

[2019-02-13 13:15:07] request.INFO: Matched route "mautic_core_ajax". {"route_parameters":{"_controller":"Mautic\\CoreBundle\\Controller\\AjaxController::delegateAjaxAction","_route":"mautic_core_ajax"},"request_uri":"https://domain/s/ajax?action=updateUserStatus&mauticLastNotificationId=&mauticUserLastActive=1"} []
[2019-02-13 13:15:07] security.DEBUG: Read existing security token from the session. {"key":"_security_mautic"} []
[2019-02-13 13:15:07] security.DEBUG: User was reloaded from a user provider. {"username":"mis","provider":"Mautic\\UserBundle\\Security\\Provider\\UserProvider"} []
[2019-02-13 13:15:07] security.DEBUG: Stored the security token in the session. {"key":"_security_mautic"} []
[2019-02-13 13:15:07] request.INFO: Matched route "mautic_contact_action". {"route_parameters":{"_controller":"Mautic\\LeadBundle\\Controller\\LeadController::executeAction","objectId":"36","objectAction":"edit","_route":"mautic_contact_action"},"request_uri":"https://domain/s/contacts/edit/36?mauticLastNotificationId=&mauticUserLastActive=1"} []
[2019-02-13 13:15:07] security.DEBUG: Read existing security token from the session. {"key":"_security_mautic"} []
[2019-02-13 13:15:07] security.DEBUG: User was reloaded from a user provider. {"username":"mis","provider":"Mautic\\UserBundle\\Security\\Provider\\UserProvider"} []
[2019-02-13 13:15:08] security.DEBUG: Stored the security token in the session. {"key":"_security_mautic"} []

And this is what I get from the ssl_access_log

IP Address - - [13/Feb/2019:13:18:42 +0100] "POST /s/ajax?action=updateUserStatus&mauticUserLastActive=1&mauticLastNotificationId= HTTP/1.1" 200 13
IP Address  - - [13/Feb/2019:13:18:44 +0100] "POST /s/contacts/edit/36?mauticUserLastActive=1&mauticLastNotificationId= HTTP/1.1" 200 539406

So, nothing strange on both log files.
Could you please help me?

Thank you and best regards,

Daniel

**Comment by escopecz** [comment]
Sadly, there is no error message that could cause such problem. What exactly happens that you've been unable to create or update any contact? Do you see 500 error when you try it? Or a timeout? Check the Mautic logs again if there is some related error message. Not just info and debug messages.

**Comment by dsalvat-cells** [comment]
Thanks @escopecz for your reply.

There is no error message, I agree. We've done some research here, and installed a fresh instance of mautic 2.15.0.

When using the DB that is created during the installation, everything works fine. But, if we change the DB and point to the upgraded one, the error can be reproduced.

We've done a diff search of both schemas, and the only differences we can see are these ones:

```
Table `campaigns` was modified
  columns:
  - modified column allow_restart
  __
Table `emails` was modified
  columns:
  - modified column variant_sent_count
  - modified column variant_read_count
  __
Table `leads` was modified
  columns:
  - added column language of type VARCHAR(255)
  __
  indices:
  - added index language_search with columns: language
```

We are inclined to think that either the 'leads' table has something to do with it or we have a data problem. 

The next step would be applying the 'leads' alter table script to a copy of the fresh db and see how it behaves. 

I'm guessing there is no one else is having this problem, isn't there?

**Comment by escopecz** [comment]
This may happen when the upgrade fails in the middle. There are ways how to fix the original database. See https://johnlinhart.com/blog/uh-oh-mautic-upgrade-was-not-successful . But the error message should help discover the problem faster. It must be somewhere in the logs.

I'll close this ticket as the tutorial above should help. If not, comment on this and I'll re-open.

**Comment by BillyHilly** [comment]
Hey @escopecz 
I just started having this issue, running v.2.15.3 on php 7.2.2 with updated schema.
Possibly related to #8283 as they both started occurring at the same time—or at least I noticed them both occurring at the same time.
Exact same process to reproduce the bug, and as @dsalvat-cells said, there's nothing in the logs. I checked the Mautic log both in Mautic an directly on the server for good measure, along with PHP logs and nothing is showing.
The typical, "Contact Updated!" pop-up notification never shows, just the line moving like something's happening then nothing.
Ironically, I can update an older contact from a previous version prior to upgrade.
And form submissions work / adds the contact, you just can't update them directly in Mautic afterwards.
I've only got 3 known contacts in this instance, all me.
I can set you up with a login if you'd like to see this issue action.
I'm not running a dev version of mautic, but I'd be happy to do what's needed to produce an error somehow.
